core/src/main/java/org/infinispan/commands/control/LockControlCommand.java

```diff
@@ -49,14 +47,14 @@ public class LockControlCommand extends AbstractTransactionBoundaryCommand imple
 
    public static final int COMMAND_ID = 3;
 
-   private List<Object> keys;
+   private Collection<Object> keys;
```

Comment:
can't we keep the List?

Reply:
We can, but it means changing the constructor as we're no longer creating the list there. As there's no use of List vs Collection, I thought this was the path of least resistance.